lib/service/database.js

```diff
@@ -87,20 +87,22 @@ DatabaseService.prototype.getMetadataFromAffectedTables = function(node, skip, c
         var defaultLastUpdatedTime = new Date('1970-01-01T00:00:00.000Z');
         var affectedTables = [];
         var lastUpdatedTimes = [];
-        var rows = (Array.isArray(resultSet.rows) && resultSet.rows.length > 0) ?
-            resultSet.rows :
-            [{updated_at: defaultLastUpdatedTime, schema_name: null, table_name: null}];
+        var rows = Array.isArray(resultSet.rows) ? resultSet.rows : [];
```

Comment:
With `var rows = resulSet.rows || []` if `resulSet.rows` is not something falsy (i.e: true) next line will try to call `forEach` and it will BOOMMM!